### DISCUSSION TITLE
Polish Payment Gateways

### DISCUSSION THREAD

**Discussion by dawidurbanski** [wpappointments/wpappointments#76] [Ideas]
Przelewy24 and PayU for the start

**Comment by tomrobak** [comment]
if WooCommerce feature will be implemented as core option then payment gateways will be worldwide

**Reply by deaf-potato** [reply]
Payment processing through woocommerce should be an option and not a simple go-to. Woocommerce loads a stack of scripts and styles on unused pages and slows down the site unnecessarily. Woocommerce It's a total unnecessary and heavy plugin for any manicurist, hairdresser, doctor and basically anyone who just needs to make appointments for people. I don't think people who create company websites need another plugin that works the same as any other. What we need is to minimize the number of unnecessary plug-ins and unnecessary features

**Reply by tomrobak** [reply]
my idea is to have woocommerce on click as a core function, not as another addon. so basically if you want to use it, click and integration loads... if not, keep it off. simple switch

**Comment by wppoland** [comment]
We have decided to go with WooCommerce as an optional plugin - not baked into core. If you enable it, booking creates a WooCommerce order and payment flows through your store gateway. If you do not need WooCommerce, it stays completely off. No extra scripts, no overhead. This way Polish gateways (Przelewy24, PayU) work automatically through the WooCommerce ecosystem.
